src/Models/Services/Utils.ts

```diff
@@ -390,6 +390,23 @@ export async function parseModel(modelJson: string) {
     }
 }
 
+export async function parseModels(models) {
```

Comment:
Can we remove the old `parseModel` function from this file?

Reply:
Can we type `models`? It should be the same type as what's on the reducer. Array of `IDtdlInterface` from memory?

Reply:
I believe `JSONEditor` file also uses `parseModel`

Reply:
Hmmm. What happens if you use the JSONEditor with a model that has an extends/component? Will it throw an error given the references won't resolve?

Reply:
In the case of only having Model0, when updating it via JSONEditor, by adding an extends
` "extends": [
    "dtmi:com:example:model2;1"
  ]`
  
We get the error `No getModels provided to resolve requisite reference(s): dtmi:com:example:model2;1`

Reply:
Ah. So I think we probably need to update JSONEditor to use `parseModels` too (i.e. pass in [...models, ...freshlyEditedModelFromJsonEditor]`).